test/com/dataengi/crm/identities/daos/UsersDAOSpec.scala

```diff
@@ -24,8 +24,9 @@ class UsersDAOSpec
     )
   }
 
-  val TestLoginInfo: LoginInfo = LoginInfo("provider", "dasd@asd.com")
-  lazy val usersDAO            = application.injector.instanceOf[UsersDAO]
+  val TestLoginInfo: LoginInfo  = LoginInfo("provider", "dasd@asd.com")
+  val TestLoginInfo2: LoginInfo = LoginInfo("provider2", "dasd2@asd.com")
```

Comment:
Maybe it will be better generating `TestLoginInfo*` in each test case. Not same for all cases